dmrunner/runner.py

```diff
@@ -245,6 +245,7 @@ def _populate_multiprocessing_components(self) -> None:
         for repository_name in itertools.chain.from_iterable(self._app_repositories):
             app_name = self.settings["repositories"][repository_name]["name"]
 
+            # BUG: is this dict used anywhere?
```

Comment:
Nope, looks like old code that wasn't removed when I created `get_app_info`. Good spot.

Reply:
Cool, I'll remove it